libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/api/ApolloGradleToolingModel.kt

```diff
@@ -22,6 +25,48 @@ interface ApolloGradleToolingModel {
     val endpointHeaders: Map<String, String>?
   }
 
+  interface TelemetryData {
+    val gradleVersion: String?
+    val androidMinSdk: Int?
+    val androidTargetSdk: Int?
+    val androidCompileSdk: String?
+    val androidAgpVersion: String?
+    val apolloServiceTelemetryData: List<ServiceTelemetryData>
+    val apolloGenerateSourcesDuringGradleSync: Boolean?
+    val apolloLinkSqlite: Boolean?
+    val apolloUseAntlr: Boolean?
```

Comment:
Late to the party but heads up this one is probably going away soon. Might not even be worth including it. Your call.

Reply:
👍 I'll remove it